src/DependencyInjection/Compiler/AddResourcesPathsPass.php

```diff
@@ -41,7 +41,7 @@ private function getResourcesPath(ContainerBuilder $container)
         $rootDir = dirname($container->getParameter('kernel.root_dir'));
 
         foreach ($container->getParameter('kernel.bundles') as $name => $class) {
-            if ('Contao\CoreBundle\HttpKernel\Bundle\ContaoModuleBundle' === $class) {
+            if (is_a($class, 'Contao\CoreBundle\HttpKernel\Bundle\ContaoModuleBundle', true)) {
```

Comment:
> It has been un-deprecated in PHP 5.3 or so…

You are right.

> neither will `instanceof` work on a string.

I don't understand this argument. `is_a()` requires an object as first argument, so it will not work on a string, either.